howto/anbox/harden.md

```diff
@@ -0,0 +1,71 @@
+(howto-harden)=
+# How to harden your deployment
+
+Anbox Cloud is designed with secure development practices but there are certain aspects to consider on your side to make sure the entire deployment is secure.
+
+## General guidelines
+
+Consider the following simple yet impactful measures to ensure that you run a secure Anbox Cloud deployment:
+
+- Always run the latest supported version of Anbox Cloud and keep your deployment up to date. See {ref}`ref-release-notes`.
+- Make sure your deployment uses machines running a supported Ubuntu version. See {ref}`ref-requirements`.
+- Do not disable TLS pinning when you are not using a load balancer.
```

Comment:
This is rather unspecific, shouldnt we put this into context of the instance to stream agent communication?